src/JuliusSweetland.OptiKey/Resources/Icons/SVG Source Files/CommuniKateToppage.svg

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Could you please change the casing of the filename (and any code uses of "Toppage") to "TopPage"